### PR TITLE
[nnpackage_run] Introduce load and dump raw

### PR DESCRIPTION
**Pull request by glistening**
It introduces reading and writing raw data for input and output respectively.

Two options are added:
--load:raw your_input_to_load_in_raw
--dump:raw your_output_dump_in_raw

RawFormatter is introduced
Formatter will also introduced as base class for H5Formatter and RawFormatter.

TODO: Support multiple raw inputs/outputs

Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

Related: #8820

### CODE REVIEW

**Comment by chunseoklee** [comment]
(optional) how about adding note "TODO: support multiple raw inputs/outputs" in commit msg ?

**Review by chunseoklee — DISMISSED** [review]

**Comment by glistening** [comment]
@chunseoklee Updated. I can finish `TODO` after we choose option 1 or option 2 in my comment. I prefer option 2. : )

File: tests/tools/nnpackage_run/src/rawformatter.cc [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+{
+void RawFormatter::loadInputs(const std::string &filename, std::vector<Allocation> &inputs)
+{
+  uint32_t num_inputs;
+  NNPR_ENSURE_STATUS(nnfw_input_size(session_, &num_inputs));
+
+  // TODO: Support multiple inputs
+  // Option 1. Get comman-separated input file list like --load:raw in.0,in.1,in.2
+  // Option 2. Get prefix --load:raw out
+  //           Internally access out.0, out.1, out.2, ... out.{N} where N is determined by api.
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/rawformatter.cc:35]: I prefer option 2.

**Review by hseok-oh — APPROVED** [review]

**Comment by glistening** [comment]
@chunseoklee PTAL

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**